src/mdns/mdns_avahi.cpp

```diff
@@ -829,6 +870,44 @@ otbrError PublisherAvahi::UnpublishHost(const char *aName)
     return error;
 }
 
+otbrError PublisherAvahi::TxtListToAvahiStringList(const TxtList &   aTxtList,
+                                                   AvahiStringList * aBuffer,
+                                                   size_t            aBufferSize,
+                                                   AvahiStringList *&aHead)
+{
+    otbrError        error = OTBR_ERROR_NONE;
+    size_t           used  = 0;
+    AvahiStringList *last  = nullptr;
+    AvahiStringList *curr  = aBuffer;
+    aHead                  = nullptr;
```

Comment:
nit: empty line after declaration block
```suggestion
    AvahiStringList *curr  = aBuffer;

    aHead                  = nullptr;
```